docs/ref/contrib/sitemaps.txt

```diff
@@ -408,11 +408,11 @@ parameter to the ``sitemap`` and ``index`` views via the URLconf::
     from django.contrib.sitemaps import views
 
     urlpatterns = [
-        url(r'^custom-sitemap\.xml$', views.index, {
+        path('custom-sitemap.xml', views.index, {
             'sitemaps': sitemaps,
             'template_name': 'custom_sitemap.html'
         }),
-        url(r'^custom-sitemap-(?P<section>.+)\.xml$', views.sitemap, {
+        path('custom-sitemap-<section>.xml', views.sitemap, {
             'sitemaps': sitemaps,
```

Comment:
The indentation here (lines 412, 413 & 416) are not the same as the others (see line 363 for example).

Reply:
The difference is that there's an open dictionary here.

Reply:
Oh! My bad! Sorry about that. It seemed weird, but it's completely normal.